src/app/services/data/data.service.ts

```diff
@@ -150,4 +155,19 @@ export class DataService {
         return observableOf(data);
       }));
   }
+
+  private getHeader(headers?) {
+    const default_headers = {
```

Comment:
@santoshilimi Don't pass it headers in the `data.services.ts` file.  Please check for another solution provide by angular.

Reply:
Updated with changes